x/epoching/keeper/msg_server.go

```diff
@@ -130,6 +132,8 @@ func (ms msgServer) WrappedStakingUpdateParams(goCtx context.Context, msgWrapped
 
 	ms.EnqueueMsg(ctx, queuedMsg)
 
+	ctx.GasMeter().ConsumeGas(ms.GetParams(ctx).EnqueueGasFees.StakingUpdateParams, "epoching staking update params enqueue fee")
```

Comment:
UpdateParams is a msg submitted through gov, so please re-examine whether enqueueing and gas consumption are actually necessary